certified-connectors/Azure Communication Services Email/apiDefinition.swagger.json

```diff
@@ -239,6 +221,15 @@
                 }
               }
             }
+          },
+          "default": {
```

Comment:
@lei-msft @alvinjiang-msft Is it possible to provide valid expected response codes with schema definitions for all the operations in the swagger, as the default responses are not allowed, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. 

**This applies to all the default responses in the swagger.**

Reply:
This action does have a 200 response on line 209, which provides dynamic content in the output.
This default response is for error code. Should we remove this default response instead?

Reply:
@lei-msft @alvinjiang-msft Yes, we need to provide only the possible and expected response codes as default response codes are not allowed anymore. Instead, you can remove the default responses to avoid confusion in the in the further review with the engineering team which would happen before prepping your connector for production deployment.